### ISSUE TITLE
Time zone for checkout should be set to EST, rather than UTC

### ISSUE THREAD

**Issue by mcailler** [sleepepi/library#2]

**Comment by saule-bussing** [comment]
fixed in 5204856a13c4e3558b73ad5423f13268cd484f73

**Comment by remomueller** [comment]
We'll need a more robust fix for this in the next update:

`config/application.rb`

``` ruby


    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
    # config.time_zone = 'Central Time (US & Canada)'
    config.time_zone = 'Eastern Time (US & Canada)'

```
